### PR TITLE
Promote Helm installation to alpha

### PR DESCRIPTION
**Pull request by nrjpoddar**
Integration tests have been to master and release-1.8 to support this promotion.

Addressed suggestions made in https://github.com/istio/istio.io/issues/8374

[ ] Configuration Infrastructure
[x] Docs
[x] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by nrjpoddar** [comment]
Helm integration tests have been implemented and running successfully in [master](https://github.com/istio/istio/tree/master/tests/integration/helm and [release-1.8](https://github.com/istio/istio/tree/release-1.8/tests/integration/helm) branches.

File: content/en/docs/setup/install/helm/index.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ test: no
 ---
 
 {{< warning >}}
-This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
-considered `pre-alpha`.
+This feature is currently considered alpha.
 {{< /warning >}}
```
Comment (frankbu) [inline content/en/docs/setup/install/helm/index.md:15]: Why do you need a warning here? Usualluy we only warn about features that are pre-alpha, subject to major change.

Comment (nrjpoddar) [reply content/en/docs/setup/install/helm/index.md:15]: I see. I wanted to make it clear that Helm support is not beta yet so they can make installation choices based on all the data. How should I do that?

**Comment by ericvn** [comment]
Do we want to add something here: https://preliminary.istio.io/latest/about/feature-stages/?

**Comment by ericvn** [comment]
I think the istio/enhancements templates become active for 1.9, but haven't checked them.

File: content/en/docs/setup/install/helm/index.md (hunk 1 above)
Comment (frankbu) [reply content/en/docs/setup/install/helm/index.md:15]: Maybe just leave the sentence in the doc, but without the `{{< warning }}` wrapper. Warning seems a little too alarmist :)

Comment (nrjpoddar) [reply content/en/docs/setup/install/helm/index.md:15]: That's fair, I will update it.

**Comment by nrjpoddar** [comment]
@frankbu or @ericvn can I get approval?

**Review by frankbu — APPROVED** [review]

**Pull request merged**